HarpGuru/HarpFace/HarpCell/index.tsx

```diff
@@ -51,17 +51,22 @@ export const HarpCell = (props: HarpCellProps): React.ReactElement => {
   }
 
   const accessibleContent = 
-    <TouchableOpacity
+    <TouchableWithoutFeedback
```

Comment:
WithoutFeedback is only acceptable because we will be adding style soon to highlight active cells which are toggled when clicked.

Reply:
This should be changed to TouchableHighlight once we have a way to test the changed state, rather than the callback function. Issue raised: https://github.com/js-jslog/harpnative/issues/8